overrides.json

```diff
@@ -26,12 +26,12 @@
           "import": "./esm/index.js"
         },
         "./index": null,
-        "./esm/*": null,
-        "./es/*": null,
         "./*": {
           "require": "./*/index.js",
           "import": "./esm/*/index.js"
-        }
+        },
+        "./esm/*": "./esm/*/index.js",
+        "./es/*": "./es/*/index.js"
```

Comment:
Do we really need both of these, can we not just expose eg `esm` which seems to be more popular?

Reply:
tbh, I've no idea, `es/` is actually documented here... https://material-ui.com/guides/minimizing-bundle-size/#ecmascript ... so i guess there is the possibility that people use it.

Personally I wasn't aware of either of these before this, but i get the feeling the packages I should be resolving to `esm/` according to the rest of the doc ... but `esm/` support appears to be incomplete on jspm cdn atm.

Reply:
Right, I guess there's no harm in keeping it.